libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -666,9 +666,9 @@ inline void JSONNode::set_name(const json_string & newname) json_nothrow{
 	   return static_cast<json_string>(*internal);
     }
 
-    inline int JSONNode::as_int(void) const json_nothrow {
+    inline json_int_t JSONNode::as_int(void) const json_nothrow {
```

Comment:
no it doesn't. These are two different concepts. The `streamdiff` is essentially a pointer on the storage stream of the repo. Where as the `JSONNode::as_int` points to the index of the `JSONNode` object that we use as metadata. If we use a different data type here than this only effects the number of objects we can store, but not the amount of data.